packages/api/src/api/expoJudgingSession/id/continue/index.ts

```diff
@@ -0,0 +1,6 @@
+import { Router } from 'express';
+import { get } from './get';
+
+export const Continue = Router();
```

Comment:
Let's lowercase the `continue` export so it matches other exported routes. If you do a rename in the editor it should fix it across all of your changes here 👍🏻

Reply:
i tried doing that it gives an error " 'continue' is not allowed as a variable declaration name."